Sources/MultipeerKit/Internal API/Extensions/MCPeerID+Me.swift

```diff
@@ -26,7 +26,7 @@ extension MCPeerID {
 
 }
 
-#if os(iOS) || os(tvOS)
+#if os(iOS) || os(tvOS) || os(visionOS)
```

Comment:
Should we just change this to `#if canImport(UIKit)`? 🤔

Reply:
Yes, this also works. Changed to `canImport`